### DISCUSSION TITLE
Poszukuje kontaktu do osoby która posiada konto służbowe w usłudze Office 365

### DISCUSSION THREAD

**Discussion by portsup13344** [AzureCommunityPL/group#217] [Pomoc]
Poszukuje kontaktu do osoby która posiada konto służbowe w usłudze Office 365,

Mam problem z otwarciem jednego linku do Office 365. Jest napisane, że tylko ze służbowym kontem mogę to zrobić. Czy jest ktoś kto mógłby sprawdzić czy mu się otworzy ten link?

**Comment by aryll** [comment]
Magiczne linki brzmią podejrzanie 😉

@portsup13344  Jak nie ma chętnych zawsze możesz założyć darmową subskrypcję Azure i otworzyć swoim użytkownikiem ten link :) - chyba że faktycznie potrzebne jest do tego całe Office 365.

Natomiast linki do Office 365 raczej tworzone są w danym tenancie/organizacji moim zdaniem więc tutaj chyba warto zacząć od odpowiedzi na pytanie co to za link 😊
